week2/new Program/index.js

```diff
@@ -0,0 +1,108 @@
+"use strict"
+/****  it's a program that do the following:
+These are the specs for this week's assignment:
+  - The user can run a NodeJs to-do app
+  - The user can be able to run the file using node index.js
+  - There should be a "help" section that lists all the commands for how to use the app
+
+The following commands should be present:
+  - No command: show help section (node index.js)
+  - help: show help section (node index.js help)
+  - list: show current todo's, or show an appropriate text if there are no todos (node index.js list)
+  - add: add a todo item. all the words behind "add" are entered as 1 todo item to the list (node index.js add "Buy groceries")
+  - remove: remove a todo item by its 1-base index. (node index.js remove 2)
+  - reset: remove all todo items from the list (node index.js reset)
+  - BONUS: update: update a todo item with new text (node index.js update 3 "Wash teeth")
+*/
+const FS = require("fs")
+const REQUIRE_ARGS = process.argv.slice(2)
+const FIRST_INPUT_COMMAND = REQUIRE_ARGS[0]
```

Comment:
Constants with capital letters aren't usually used for user input. It typically signifies that there's a "magic value" somewhere in the code, so it would be changed by changing code somewhere - but because it's a constant, we can reuse the constant everywhere instead of having to copy the magic value.

You can still use `const`, since it's not going to change while the program is running - but I wanted to explain a bit about the idiom of the capital letters thing, since it looks weird to me here.

Reply:
looking forward For that 🙂